content/advocacy/mvmha/index.md

```diff
@@ -29,22 +29,32 @@ The MVMHA includes all six mobile home communities in Mountain View:
  
-On September 14, 2021, City Attorney Krishan Chopra presented a draft of a proposed ordinance to the City Council, and on September 15, the Mountain View City Council voted 6-1 to enact rent control for mobile homes. After fighting for many years, mobile home residents now have protections against rent hikes and protection from other predatory practices by park owners. Our space rents were rolled back to March 16, 2021, and the annual increases were only allowed to resume on September 1, 2022.
+On September 14, 2021, City Attorney Krishan Chopra presented a draft of a proposed ordinance to the City Council. On September 15, the Mountain View City Council voted 6-1 to enact rent control for mobile homes.
+
+### What is the Status of MHRSO Today?
+
+After fighting for many years, mobile home residents now have protections against rent hikes and protection from other predatory practices by park owners. Our space rents were rolled back to March 16, 2021, and the annual increases were only allowed to resume on September 1, 2022.
  
-But that's not the end of the story. One of the park owners fought back with an ill-conceived alternative, a Memorandum of Understanding, that City Council finally voted down on June 22, 2022.
+But that wasn't the end of the story. One of the park owners fought back with an ill-conceived alternative, a Memorandum of Understanding, which would have added an exemption to rent control for one of Mountain View's parks. But on June 22, 2022, City Council voted to remove exemptions from the Ordinance, and now residents of all six mobile home parks in Mountain View are finally protected by the MHRSO. 
+
+### What is the Next Step?
  
-Now we're fighting for a lower ceiling to rent increases (it's now 5%), and we have gotten a commitment from City Council that they will amend our Ordinance by March of 2025. 
+Now we're fighting for a lower ceiling to rent increases (it's now 5%), and after adding our program to the City's Housing Element, which was certified by California's Department of Housing and Community Development on May 26, 2023, we got a commitment from City Council that they will study and amend our Ordinance by March of 2025. 
 
-Then we're going to fight to own the land under our homes by learning how we might convert our parks to resident-owned communities. That's going to be another significant, long-term struggle - but it will be worth it if we can find funding from a community land trust to help our residents own their spaces as well as their homes!
\ No newline at end of file
+ We're also going to fight to own the land under our homes by learning how we might convert our parks to resident-owned communities. That's going to be another significant, long-term struggle - but it will be worth it if we can find funding from a community land trust to help our residents own their spaces as well as their homes.
```

Comment:
```suggestion
We're also going to fight to own the land under our homes by learning how we might convert our parks to resident-owned communities. That's going to be another significant, long-term struggle - but it will be worth it if we can find funding from a community land trust to help our residents own their spaces as well as their homes.
```

Reply:
change already made